templates/travel/elements/_date-input.css

```diff
@@ -0,0 +1,85 @@
+.travel-date-input {
+  font-weight: 600;
+  line-height: var(--input-line-height);
+  padding: var(--input-padding-y) var(--input-padding-x);
+  padding-left: calc(var(--input-padding-y) * 2.5 + var(--icon-size));
+  position: relative;
+}
+
+.travel-date-input .travel-icon{
+  position: absolute;
```

Comment:
God catch. As I review this PR again I'll look out for moving properties to basscss classes. I think in this case these properties are declared this way so that the ```.travel-icon``` is absolutely positioned only when it's a child of ```.travel-date-input```